package/MDAnalysis/core/universe.py

```diff
@@ -739,7 +769,7 @@ def coord(self):
         represented as a :class:`numpy.float32` array.
 
         Because :attr:`coord` is a reference to a
-        :class:`~MDAnalysis.coordinates.timestep.Timestep`, it changes its contents
+        :class:`~MDAnalysis.coordinates.base.Timestep`, it changes its contents
```

Comment:
```suggestion
        :class:`~MDAnalysis.coordinates.timestep.Timestep`, it changes its contents
```

I know I told you `base` at some point but it looks like I got it wrong: `base` imports `Timestep` from `timestep`. https://github.com/MDAnalysis/mdanalysis/blob/81ed5d0073d5eae032790bb6ced7fc217e994d3a/package/MDAnalysis/coordinates/base.py#L132